zgw/catalogi-api/src/main/kotlin/com/ritense/catalogiapi/client/CatalogiApiClient.kt

```diff
@@ -202,6 +227,30 @@ class CatalogiApiClient(
         return result?.body!!
     }
 
+    open fun prefillCache(authenticationPluginConfiguration: CatalogiApiAuthentication, url: URI) {
+        prefillInformatieobjecttypeCache(authenticationPluginConfiguration, url)
+    }
+
+    private fun prefillInformatieobjecttypeCache(authenticationPluginConfiguration: CatalogiApiAuthentication, url: URI) {
+        var currentPage = 1
```

Comment:
Please take a look at the `com.ritense.zgw.Page.getAll(...)` helper method.

Reply:
Fixed